specs/packagemanager/PackageManagement.md

```diff
@@ -0,0 +1,1136 @@
+
+        /// Get the specified volume.
+        /// @see https://learn.microsoft.com/en-us/uwp/api/windows.management.deployment.packagemanager.findpackagevolume
+        static PackageVolume FindPackageVolumeByPath(String path);
+
+        /// Get the specified volume.
+        /// @see https://learn.microsoft.com/en-us/uwp/api/windows.management.deployment.packagemanager.findpackagevolume
+        static PackageVolume FindPackageVolumeByMediaId(String mediaId);
+
+        /// Get the specified volume.
+        /// @see https://learn.microsoft.com/en-us/uwp/api/windows.management.deployment.packagemanager.findpackagevolume
+        static PackageVolume FindPackageVolumeByName(String name);
+    };
+
+    /// The progress status of the deployment request.
+    /// @see https://learn.microsoft.com/en-us/uwp/api/windows.management.deployment.deploymentprogress.state?view=winrt-22621
+    [contract(PackageDeploymentContract, 1)]
+    enum PackageDeploymentProgressStatus
+    {
+        Queued = 0,             // The request is queued
+        InProgress = 1,         // The request is in progress
+        CompletedSuccess = 2,   // The request completed successfully
+        CompletedFailure = 3,   // The request failed with some critical internal error.
+    };
+
+    /// Contains progress information for the deployment request.
+    /// @see https://learn.microsoft.com/en-us/uwp/api/windows.management.deployment.deploymentprogress?view=winrt-22621
+    [contract(PackageDeploymentContract, 1)]
+    runtimeclass PackageDeploymentProgress
+    {
+        PackageDeploymentProgressStatus status;
+
+        /// The progress percentage of the deployment request.
+        /// @note This is a double with values 0.0-1.0. Windows.Management.Deployment.DeploymentProgress.percentage is uint32 with values 0-100.
+        Double percentage;
+    }
+
+    /// The status of the deployment request.
+    /// @see PackageDeploymentResult.Status
+    [contract(PackageDeploymentContract, 1)]
+    enum PackageDeploymentStatus
+    {
+        InProgress = 0,         // The request is in progress
+        CompletedSuccess = 1,   // The request completed successfully
+        CompletedFailure = 2,   // The request failed with some critical internal error.
+    };
+
+    // Provides the result of a deployment request.
+    // @see https://learn.microsoft.com/en-us/uwp/api/windows.management.deployment.deploymentresult
+    [contract(PackageDeploymentContract, 1)]
+    runtimeclass PackageDeploymentResult
+    {
+        PackageDeploymentStatus Status { get; };
+        HRESULT ExtendedError { get; };
+        Boolean IsRegistered{ get; };
+        Guid ActivityId { get; };
+    }
+
+    [contract(PackageDeploymentContract, 1)]
+    runtimeclass PackageSetItem
+    {
+        PackageSetItem();
+
+        String Id;
+
+        // Package criteria to identify if a matching package is ready for use and to use at runtime
+        String PackageFamilyName;   //required
```

Comment:
These comments are specific to using PackageSet in packageRuntimeManager.AddPackageSet or packageRuntimeManager.RemovePackageSet.  They are not true when using PackageSet for other operations like AddPackageSetAsync.

Reply:
Comments are holdover from early implementation. Obsolete now so I've removed them